src/TaylorBot.Net/Program.Commands.Discord/src/TaylorBot.Net.Commands.Discord.Program/Modules/DiscordInfo/Commands/AvatarSlashCommand.cs

```diff
@@ -12,14 +12,21 @@ public class AvatarCommand
     {
         public static readonly CommandMetadata Metadata = new("avatar", "DiscordInfo 💬", new[] { "av", "avi" });
 
-        public Command Avatar(IUser user, string? description = null) => new(
+        public Command Avatar(IUser user, string? description = null, string? type = "guild") => new(
             Metadata,
             () =>
             {
+                type ??= "guild";
+                var avatarUrl = user is IGuildUser guildUser ?
```

Comment:
I think we should add a new extension `GetGuildAvatarUrlOrDefault()` and consider changing using it in other commands as well